docs/contribute/topics.md

```diff
@@ -75,21 +75,62 @@ $ pre-commit run --all-files
 $ pre-commit run -a
 ```
 
-## Web asset compiling and bundling
+## Web assets (CSS/JS/Fonts)
 
-All of the CSS and JS assets stored in `src/pydata_sphinx_theme/assets` will be compiled and bundled with the theme when you build it locally.
-These bundled assets will be placed in `src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static`.
+This theme includes several web assets to ease development and design.
+The configuration for our asset compilation is in `webpack.config.js`.
 
-The configuration that defines what happens upon compilation is at `webpack.config.js`.
+### Compile and bundle assets
 
-When assets are compiled, a `<hash>` is generated for each, and appended to the end of the asset's reference in the HTML templates of the theme.
+When assets are compiled, static versions are placed in various places in the theme's static folder:
+
+```
+src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static
+```
+
+For many assets, a `<hash>` is generated and appended to the end of its reference in the HTML templates of the theme.
 This ensures the correct asset versions are served when viewers return to your
 site after upgrading the theme.
 
-Web fonts, and their supporting CSS, are copied into
-`src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static/vendor/<font name>/<font version>/`.
+To compile the assets and bundle them with the theme, run this command:
+
+```console
+$ nox -s compile
+```
+
+### Styles (SCSS) and Scripts (JS)
+
+There are two relevant places for CSS/JS assets:
+
+- `src/pydata_sphinx_theme/assets/styles` has source files for SCSS assets. These will be compiled to CSS.
+- `src/pydata_sphinx_theme/assets/scripts` has source files for JS assets. These will be compiled to JS and import several vendored libraries (like Bootstrap).
+- `src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static` has compiled versions of these assets (e.g. CSS files). This folder is not tracked in `.git` history, but it is bundled with the theme's distribution.
 
-The links to these unique file names are captured as Jinja2 macros that are defined in HTML templates bundled with the theme.
+### Vendored scripts
+
+We vendor several packages in addition to our own CSS and JS.
+For example, Bootstrap, JQuery, and Popper.
+This is configured in the `webpack.config.js` file, and imported in the respective `SCSS` or `JS` file in our assets folder.
+
+### FontAwesome icons
+
+Three "styles" of the [FontAwesome 5 Free](https://fontawesome.com/icons?m=free)
+icon font are used for {ref}`icon links <icon-links>` and admonitions, and is
+the only `vendored` font.
+
+- It is managed as a dependency in `package.json`
+- Copied directly into the site statics at compilation, including licenses
+- Partially preloaded to reduce flicker and artifacts of early icon renders
+- Configured in `webpack.js`
```

Comment:
```suggestion
- Configured in `webpack.config.js`
```
The original sentence was referencing the webpack.js package but here I think you want to reference the webpack config file, right?